app/views/ems_physical_infra/_main.html.haml

```diff
@@ -0,0 +1,11 @@
+= render :partial => "layouts/flash_msg"
+.row
+  .col-md-12.col-lg-6
```

Comment:
I believe the Lenovo guys did test the Textual Summary page, and AFAIK, everything seemed to work OK. 
But we'll address the obsolete part.